common/pb/response.proto

```diff
@@ -1,58 +1,59 @@
 message Response {
```

Comment:
Reformatting the whole file here in the same commit as editing it makes it hard to review.